packages/next/src/lib/metadata/generate/basic.tsx

```diff
@@ -12,14 +12,17 @@ export function BasicMetadata({ metadata }: { metadata: ResolvedMetadata }) {
       ) : null}
       <Meta name="description" content={metadata.description} />
       <Meta name="application-name" content={metadata.applicationName} />
-      {metadata.authors &&
-        metadata.authors.map((author, index) => (
-          <React.Fragment key={index}>
-            {author.url && <link rel="author" href={author.url.toString()} />}
-            <Meta name="author" content={author.name} />
-          </React.Fragment>
-        ))}
-      <Meta name="manifest" content={metadata.manifest?.toString()} />
+      {metadata.authors
+        ? metadata.authors.map((author, index) => (
+            <React.Fragment key={index}>
+              {author.url && <link rel="author" href={author.url.toString()} />}
+              <Meta name="author" content={author.name} />
+            </React.Fragment>
+          ))
+        : null}
+      {metadata.manifest ? (
+        <link rel="manifest" href={metadata.manifest?.toString()} />
```

Comment:
```suggestion
        <link rel="manifest" href={metadata.manifest.toString()} />
```
the above check makes it defined